src/db/analytics.ts

```
@@ -19,22 +32,81 @@ export class AnalyticsDAO {
-              { event: { contains: search } },
-              { data: { string_contains: search } },
-            ],
-          },
-        },
-        orderBy,
-      });
+      const { source, event, window, windowPage } = filters || {};
+      let dateRange;
+
+      if (window) {
+        dateRange = this.getDateRange(window, windowPage);
```

Biome (1.9.4)

[error] Using this in a static context can be confusing.

this refers to the class.
Unsafe fix: Use the class name instead.


(lint/complexity/noThisInStatic)